model/src/main/java/org/cbioportal/genome_nexus/model/PostTranslationalModification.java

```diff
@@ -0,0 +1,87 @@
+package org.cbioportal.genome_nexus.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.List;
+
+@Document(collection="ptm.experimental")
```

Comment:
data source at dbPTM is named as `experimental and putative PTM sites`, but still unsure if this is the best name for our db collection